dexbot/strategies/relative_orders.py

```diff
@@ -26,14 +29,33 @@ def configure(cls, return_base_config=True):
                           'Automatically adjust orders up or down based on the imbalance of your assets', None),
             ConfigElement('manual_offset', 'float', 0, 'Manual center price offset',
                           "Manually adjust orders up or down. "
-                          "Works independently of other offsets and doesn't override them", (-50, 100, 2, '%'))
+                          "Works independently of other offsets and doesn't override them", (-50, 100, 2, '%')),
+            ConfigElement('reset_on_partial_fill', 'bool', True, 'Reset orders on partial fill',
+                          'Reset orders when buy or sell order is partially filled', None),
+            ConfigElement('partial_fill_threshold', 'float', 30, 'Fill threshold',
+                          'Fill threshold to reset orders', (0, 100, 2, '%')),
```

Comment:
Maybe this should say `Order fill threshold to reset orders` to be more descriptive